bin/publish

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
You can remove this file, and add `#!/usr/bin/env lsc` as the first line if `publish.ls`.